### ISSUE TITLE
Potential for tick / cross emoji?

### ISSUE THREAD

**Issue by ceeK** [shaps80/Spry#3]
I recall seeing this on Twitter awhile back and thinking it was cool. 

It can look cleaner to use emoji for the test results in the timeline:
✅
❌

Just a thought!

**Comment by shaps80** [comment]
I considered this but in Playgrounds (particularly) on iPad I didn't want to have console output initially. So for me the 'traffic lights' are cleaner.

That said, I'm working on bringing in full Nimble atm -- so if it makes then I can imagine that being nice in the console output at least.

**Comment by ceeK** [comment]
What I mean is replacing the traffic light colours with emoji strings to remove the RGB values. So still traffic lights? 

![screen shot 2017-03-01 at 11 28 22](https://cloud.githubusercontent.com/assets/1478253/23458040/43f2ce24-fe72-11e6-8194-c66b25eb52bb.png)

Unless there is a technical limitation why strings aren't possible of course (or at least worse).

Arguably this could be less confusing to the new user than displaying R: 100, G: 0, B: 0 for test results.

**Comment by shaps80** [comment]
Ah I see... I hate those particular emoji haha but maybe you're right. I'll look into that. Will also check that iPad produces the same result.

**Comment by ceeK** [comment]
If the demographic of iPad testing is to let people new to coding get into testing, the tick and cross could further reinforce the success / failure :)

Just a thought though!

**Comment by shaps80** [comment]
True. Although I hate those particular emoji's I also like that fact that these are unlikely to clash/cutter other output. E.g. if I am _also_ outputting color values in my code. It can be somewhat confusing.

**Comment by ceeK** [comment]
For completeness I found my original inspiration for this ticket: https://github.com/Liquidsoul/XCTestPlayground

**Comment by shaps80** [comment]
Ok so @ceeK --  I don't like the clipping on those emoji so I've come up with an alternative.  
I also think it brings MORE attention to failed tests and less attention to passing ones.  
Thoughts?
<img width="498" alt="screen shot 2017-03-03 at 17 45 40" src="https://cloud.githubusercontent.com/assets/251310/23562262/a11120a6-0039-11e7-8936-513214f75e64.png">

**Comment by chrisamanse** [comment]
+1 on the emojis. It also has better accessibility (color blindness).

**Comment by shaps80** [comment]
The issue with this has been that iPad doesn't show the emoji's unfortunately – but I will definitely include this for the Xcode builds.
